tox.ini

```
@@ -38,3 +38,50 @@ commands = pre-commit run --all-files --show-diff-on-failure
 deps = twine
 commands_pre =
 commands = twine check {distdir}/*
+
+[testenv:build-docs]
+basepython = python3
+depends =
+  make-changelog
+deps =
+  -r{toxinidir}/docs/requirements.txt
+  # FIXME: re-enable the "-r" + "-c" paradigm once the pip bug is fixed.
+  # Ref: https://github.com/pypa/pip/issues/9243
+  # -r{toxinidir}/docs/requirements.in
+  # -c{toxinidir}/docs/requirements.txt
+description = Build The Docs
+commands =
+  # Retrieve possibly missing commits:
+  -git fetch --unshallow
+  -git fetch --tags
+
+  # Build the html docs with Sphinx:
+  {envpython} -m sphinx \
+    -j auto \
+    -b html \
+    {tty:--color} \
+    -a \
+    -n \
+    -W \
```

I think it makes sense to group these options because they serve the same purpose:
```suggestion
    -n -W --keep-going \
```